public/docs/ts/latest/guide/testing.jade

```diff
@@ -164,7 +181,7 @@ table(width="100%")
         End-to-end tests explore the application _as users experience it_.
         In e2e testing, one process runs the real application
         and a second process runs protractor tests that simulate user behavior 
-        and assert that the application responds in the browser as expected.
+        and assert that the application respond in the browser as expected.
```

Comment:
It should only have an "s" in the indicative though, not in the subjunctive. The subjunctive here is required by "assert", which introduces the subordinate clause of expectation: "that the application respond" and the change of subject. It is the same construction as "She insists that _he see_ a doctor." An alternative in English, which usually obscures the subjunctive, is "She wants _him to see_ a doctor." If the first clause and second clause have different subjects and there is doubt, disbelief, denial, or imposition of will (under which expectation falls), the second clause must use the subjunctive. So you get things like: 

> Mom expects that he eat his lunch.
> The law requires that the school have a reading program.

@wardbell I hope you read that in Leonard Nimoy's voice.